packages/front-end/components/GuidedGetStarted/GuidedGetStarted.tsx

```diff
@@ -170,6 +176,7 @@ export default function GuidedGetStarted({
             <FeatureModal
               inline={true}
               cta={"Next: Install an SDK"}
+              disabledMessage="You do not have permission to create a feature flag."
```

Comment:
This doesn't seem like the right place to do this.  What if we add logic inside FeatureModal that disables the CTA because of something else (e.g. a validation error)?  The FeatureModal component should be the one that determines why it's disabled.

Reply:
Updated!